cloud_edition/flexibility_mode/docs/pim_application.rst

```diff
@@ -13,55 +13,18 @@ With the SSH access, you can execute from the **/home/akeneo/pim/** application
 
     ~/pim $ bin/console
 
-3. Crontab configuration
```

Comment:
Is this intended to remove this section?

Reply:
Yep, that is the change I made in 2.3 :-).